src/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(HEADERS
 
 if(APPLE)
     list(APPEND SOURCES impl/apple/external_commit_helper.cpp impl/apple/keychain_helper.cpp)
-elseif(REALM_PLATFORM STREQUAL "Android")
+elseif(LINUX OR REALM_PLATFORM STREQUAL "Android")
     list(APPEND SOURCES impl/android/external_commit_helper.cpp)
```

Comment:
Seems surprising for a file with "android" in the path to be used on anything but Android.